cabal.project

```diff
@@ -19,5 +34,6 @@ constraints:
   hjsonpointer >= 1.5.0,
   hjsonschema >= 1.10.0
 
--- fixme: this shouldn't be required - not sure why it is
+-- fixme: the constraints above require 'hjsonpointer >= 1.5.0', but 'hjsonschema' has
+-- constraints that specify 'hjsonpointer>=1.1 && <1.4'.
 allow-newer: *
```

Comment:
I didn't mean for this "fix" to get merged.

It probably got put back accidentally after a force push of a PR branch.

Are you able to specify `allow-newer` for just the offending packages, instead of `*`?

Reply:
Done!